pkg/knowledge_base2/properties/float_property_test.go

```diff
@@ -238,7 +237,7 @@ func Test_FloatProperty_Validate(t *testing.T) {
 			for _, call := range test.mockKBCalls {
 				mockKB.On(call.Method, call.Arguments...).Return(call.ReturnArguments...)
 			}
-			ctx := knowledgebase.DynamicValueContext{
+			ctx := knowledgebase2.DynamicValueContext{
```

Comment:
nit: For consistency, this should be imported as `knowledgebase` - that's what we do everywhere else in the code.